lib/compat/wordpress-6.2/class-gutenberg-rest-global-styles-controller-6-2.php

```diff
@@ -16,7 +16,288 @@ class Gutenberg_REST_Global_Styles_Controller_6_2 extends WP_REST_Global_Styles_
 	 * @return void
 	 */
 	public function register_routes() {
-		parent::register_routes();
+		register_rest_route(
```

Comment:
The changes in the routes:

| Route                                           | Before                                                | After                                                                |
| ----------------------------------------------- | ----------------------------------------------------- | -------------------------------------------------------------------- |
| `global-styles/themes/<stylesheet>/variations/` | READABLE. Lists the variations of the active theme.   | Same.                                                                |
| `global-styles/themes/<stylesheet>/`            | READABLE. Lists the theme.json of the active theme.   | Same.                                                                |
| `global-styles/<cpt_id>/`                       | READABLE+EDITABLE. Lists and updates the user styles. | READABLE+EDITABLE+DELETABLE. List, updates, and deletes user styles. |
| `global-styles/`                                | -                                                     | READABLE+CREATABLE. Lists and creates user style variations.                                                  |